lte/gateway/python/integ_tests/s1aptests/util/traffic_util.py

```diff
@@ -32,7 +32,7 @@
 )
 
 # Tests shouldn't take longer than a few minutes
-TRAFFIC_TEST_TIMEOUT_SEC = 180
+TRAFFIC_TEST_TIMEOUT_SEC = 60
```

Comment:
same as above

Reply:
Removed the changes and will raise a separate PR for this